node/directives/helper.ts

```diff
@@ -2,12 +2,12 @@ import { isUserPartOfBuyerOrg } from '../resolvers/Queries/Users'
 
 export const validateAdminToken = async (
   context: Context,
-  adminUserAuthToken: string
+  adminUserAuthToken: string,
+  metricFields: any
```

Comment:
instead of providing an additional field here, can't we extract this info from `context`? This should reduce the amount of changes on this PR and cognitive load while reading the code.

Reply:
We can't get the `operation` field through the context. I think this is relevant to the metric so we need to pass it somehow. Maybe we could only pass this field and build the rest using the context.

Reply:
Right, in this case lets keep like this.